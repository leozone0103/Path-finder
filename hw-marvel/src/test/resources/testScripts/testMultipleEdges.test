# Test multiple edges from same two nodes

LoadGraph heroes multipleEdges.csv
ListChildren heroes Daniel
ListChildren heroes Karlie