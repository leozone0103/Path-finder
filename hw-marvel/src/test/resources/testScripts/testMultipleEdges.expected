# Test multiple edges from same two nodes

loaded graph heroes
the children of Daniel in heroes are: Karlie(book1) Karlie(book2)
the children of Karlie in heroes are: Daniel(book1) Daniel(book2)